test/metabase/test/util/log.clj

```diff
@@ -1,15 +1,16 @@
 (ns metabase.test.util.log
   "Utils for controlling the logging that goes on when running tests."
+  (:require [clojure.tools.logging :as log])
   (:import java.io.PrintStream
            [org.apache.commons.io.output NullOutputStream NullWriter]
            [org.apache.logging.log4j Level LogManager]
            [org.apache.logging.log4j.core Logger LoggerContext]))
 
 (def ^:private logger->original-level
-  (let [loggers (.. ^LoggerContext (LogManager/getContext false)
-                    (getLoggers))]
-    (into {} (for [^Logger logger loggers]
-               [logger (.getLevel logger)]))))
+  (delay
```

Comment:
TIL about `delay`

Reply:
so actually this didn't end up fixing it but I left it in because it still saves the work until we actually need to do it